### PR TITLE
Create Implementation Skeleton 

### PR DESCRIPTION
**Pull request by RobotSail**
:skull: 
Signed-off-by: Oleg <97077423+RobotSail@users.noreply.github.com>

### CODE REVIEW

File: copilot-ops/main.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
+
+*/
+package main
+
+import "github.com/redhat-et/copilot-ops/cmd"
```
Comment (guymguym) [inline copilot-ops/main.go:7]: Does this work for you ok? I thought you will need to add the `copilot-ops` subdir to imports, meaning `import "github.com/redhat-et/copilot-ops/copilot-ops/cmd"` 

I think we can move the go module to the root of the project, it is going to be the main go module of this project. WDYT?

File: copilot-ops/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+## Copilot-Ops CLI
```
Comment (guymguym) [inline copilot-ops/README.md:1]: This readme is gold. I love it.

**Review by guymguym — APPROVED** [review]
This is great. We definitely need to start a discussion on the details of `@filetag` file references, to make sure we have a good direction, because I believe it might be something we want to keep internal to the tool (at least to begin with).

In any case, nothing is a blocker, and this is a great skeleton PR.

We should followup and create a few issues to highlight the gaps to fill in this skeleton.

Fireaway and merge!

File: copilot-ops/main.go (hunk 1 above)
Comment (RobotSail) [reply copilot-ops/main.go]: @guymguym Moved the directories up & placed the command definitions under `pkg/cmd`.

**Pull request merged**